lib/cli.js

```diff
@@ -35,7 +35,15 @@ module.exports = function (args) {
     args.css.plugins = args.css.p || args.css.plugins
     args.css.compress = args.css.c || args.css.compress
 
-    if (args.css.output) css(args.css, writer(args.css.output, args.css))
+    if (args.css.output) {
+      css(args.css, function doWatchWriteServerstart() {
+        var resourcepaths       = arguments['2'];
```

Comment:
What exactly do you mean?
Passing in what as an argument in line 39?
"doWatchWriteServerstart" as a function is passed in as an argument to css (=require('atomify-css')).

I dont understand